src/ui/utils/sanitize.ts

```diff
@@ -43,12 +43,22 @@ export function sanitize(obj: any, path: string, category: string, logSanitized:
   return obj;
 }
 
+export function sanitizeState(state: any, logSanitized: boolean) {
+  return sanitize({ ...state, ast: undefined }, "", "state", logSanitized);
+}
+
+export function sanitizeAction(action: any, logSanitized: boolean) {
+  return action.type === "SET_SYMBOLS"
```

Comment:
What's this doing again? 

It seems like this will naively return an empty action for SET_SYMBOLS. Wouldn't we want to do something like

```js
if (EXCLUDED_ACTIONS.include(action.type) {
  return action
}

return sanitize(action, "", `action[${action.type}]`, logSanitized);
```

Reply:
Yes, I could just return that action - but then LogRocket would immediately complain it is too large and stop processing the redux state and actions.

Reply:
Thanks, i got confused between the two usecases. Also, didn't realize that our middleware continued to use action, which is not affected.